schemas/content/product-attributes.schema.json

```diff
@@ -0,0 +1,193 @@
+        },
+        "schema:width": {
+          "title": "Width",
+          "$ref": "#/definitions/attribute",
+          "description": "The width of the product or varinat."
+        },
+        "schema:brand": {
+          "title": "Brand name",
+          "$ref": "#/definitions/attribute",
+          "description": "The brand of the product or varinat."
+        },
+        "xdm:department": {
+          "title": "Department name",
+          "$ref": "#/definitions/attribute",
+          "description": "The department name the product or varinat belongs to."
+        },
+        "schema:manufacturer": {
+          "title": "Manufacturer name",
+          "$ref": "#/definitions/attribute",
+          "description": "The manufacturer name of the product or varinat."
+        },
+        "schema:mpn": {
+          "title": "Manufacturer Part Number",
+          "$ref": "#/definitions/attribute",
+          "description": "The Manufacturer Part Number (MPN) of the product or varinat."
+        },
+        "xdm:shippingWeight": {
+          "title": "Shipping Weight",
+          "$ref": "#/definitions/attribute",
+          "description": "The shipping weight of the product or varinat."
+        },
+        "xdm:review": {
+          "title": "Customer Reviews",
+          "$ref": "#/definitions/attribute",
+          "description": "Customer Reviews related to the product or varinat."
+        },
+        "xdm:warranty": {
+          "title": "Warranty & Support",
+          "$ref": "#/definitions/attribute",
+          "description": "Warranty & Support information of the product or varinat."
+        },
+        "xdm:upc": {
+          "title": "UPC",
+          "$ref": "#/definitions/attribute",
+          "description": "Universal Product Code of the product or varinat."
+        },
+        "xdm:ean": {
+          "title": "EAN",
+          "$ref": "#/definitions/attribute",
+          "description": "International Article Number of the product or varinat."
+        },
+        "schema:isAccessoryOrSparePartFor": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/attribute"
+          },
+          "description": "A pointer to another product (or multiple products) for which this product is an accessory or spare part."
+        },
+        "schema:isRelatedTo": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/attribute"
+          },
+          "description": "A pointer to another, somehow related product (or multiple products)."
+        },
+        "schema:isSimilarTo": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/attribute"
+          },
+          "description": "A pointer to another, functionally similar product (or multiple products)."
+        }
+      },
+      "patternProperties": {
+        ".+://.+": {
+          "$ref": "#/definitions/attribute"
+        }
+      }
+    },
+    "attribute": {
```

Comment:
Can we move this to an external file so it is easy to incorporate in other schemas please